application/views/assessment/assessments_edit.php

```diff
@@ -115,8 +115,7 @@
                       <div class='col-sm-12'>
                         <div class="htmleditor input-group">
                           <textarea name='assessmentmessage_<?=$assessmentlang?>' class="form-control" id='assessmentmessage_<?=$assessmentlang?>' rows='10' ><?php echo $message;?></textarea>
-                            <?php // todo implement new ekeditor 1580136051118 ?>
-                            <?php //echo getEditor("assessment-text","assessmentmessage_".$assessmentlang, "[".gT("Message:", "js")."]",$surveyid,$gid,null,$action); ?>
+                          <?php echo getEditor("assessment-text","assessmentmessage_".$assessmentlang, "[".gT("Message:", "js")."]",$surveyid,$gid,null,''); ?>
```

Comment:
Both ? textarear + getEditor ? If i don't make error getEditor function add textarea

Reply:
getEditor only bootstrap the htmleditor with the JS needed

Reply:
Right need both :)
https://gitlab.com/SondagesPro/OrcidAuthenticate/-/blob/master/views/admin/IntroductionSetting.php

Thanks